### PR TITLE
Update from Intelsdi-x pluginsync 0.1.8

### PR DESCRIPTION
**Pull request by katarzyna-z**

### CODE REVIEW

File: .pluginsync.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+- .travis.yml
+- CONTRIBUTING.md
+- LICENSE
+- Makefile
+- scripts
+- scripts/build.sh
+- scripts/common.sh
+- scripts/deps.sh
+- scripts/pre_deploy.sh
+- scripts/test.sh
```
Comment (katarzyna-z) [inline .pluginsync.yml]: @nanliu Despite the fact that `Makefile`, `scripts/build.sh`, `scripts/test.sh` are unmanaged and I see the following output from snap-pluginsync:
```
Not managing Makefile in snap-plugin-publisher-heka
Not managing scripts/build.sh in snap-plugin-publisher-heka
Not managing scripts/test.sh in snap-plugin-publisher-heka
```
these files are mentioned in `.pluginsync.yml`. It seems to be an issue in snap-pluginsync. What do you thing?

The full log from snap-pluginsync is available [here](https://gist.github.com/katarzyna-z/f4c3b7a9dec3531a8679accabd66b177),

File: .travis.yml [hunk 2]
```diff
@@ -1,33 +1,78 @@
...
-    - TEST=integration
-addons:
-  apt:
-    packages:
-      - debhelper # required for trusty VM
-      - cmake
-      - libgeoip-dev
-      - protobuf-compiler
+  - TEST_TYPE=small
+  - TEST_TYPE=build
```
Comment (katarzyna-z) [inline .travis.yml]: Currently this plugin supports old types of tests (unit and integration) so in this configuration tests are successfully running on travis but it is useless - nothing is tested for now.
This plugin should be adjusted to new names of tests but I think that is should be done in separate pull request. Change of tests names has an impact on several files.
Now, I can just add old types of tests to matrix. 
What do you think?

File: .travis.yml [hunk 3]
```diff
@@ -1,33 +1,78 @@
...
+    env: TEST_TYPE=build
+before_install:
+- "[[ -d $SNAP_PLUGIN_SOURCE ]] || mkdir -p $ORG_PATH && ln -s $TRAVIS_BUILD_DIR $SNAP_PLUGIN_SOURCE"
 install:
-- export TMPDIR=$HOME/tmp
-- mkdir -p $TMPDIR
-- cd $SNAP_PLUGIN_SOURCE # change dir into source
-- make
+- cd $SNAP_PLUGIN_SOURCE
+- make deps
```
Comment (katarzyna-z) [inline .travis.yml]: If we decide that old types should be added to test matrix in .travis.yml then this line should be changed to `make`, for now heka tests require building of plugin. Should I open an issue about it?

File: .pluginsync.yml (hunk 1 above)
Comment (nanliu) [inline .pluginsync.yml]: Yep, looks like the file listing in .pluginsync.yml didn't scan for unmanaged files.

File: .travis.yml (hunk 2 above)
Comment (nanliu) [inline .travis.yml]: Since we are using the only test, we can change the .sync.yml to reflect that.

File: .travis.yml (hunk 3 above)
Comment (nanliu) [inline .travis.yml]: We can patch the makefile behavior to suit our need.

**Comment by nanliu** [comment]
I'll generate a PR based on your current WIP.

**Comment by nanliu** [comment]
@katarzyna-z, see PR #23, specifically commit: https://github.com/intelsdi-x/snap-plugin-publisher-heka/pull/23/commits/619c81e1a00cff0c2beef3aa39aaf233ecb164d1 for changes that address your questions in this PR.

**Comment by katarzyna-z** [comment]
Changes addressed through #23
@nanliu Thanks for help 🌹

**Pull request closed without merging**